frontend/src/components/post-card.tsx

```diff
@@ -8,15 +8,17 @@ export default function PostCard({ post }: { post: Post }) {
   const navigate = useNavigate();
   const slug = createSlug(post.title);
   return (
-    <div className="w-full md:w-1/2 lg:w-1/3 xl:w-1/4">
+    <div className="w-full sm:w-1/2 md:w-1/2 lg:w-1/3 xl:w-1/4">
       <div
-        className="mr-8 mt-4 mb-4 cursor-pointer rounded-lg bg-light shadow-md dark:bg-dark-card"
+        className={`cursor-pointer rounded-lg bg-light shadow-md dark:bg-dark-card mb-4 ${
+          'md:mr-8 md:mt-4'
+        }`}
         onClick={() => navigate(`/details-page/${slug}/${post._id}`, { state: { post } })}
       >
         <img
           src={post.imageLink}
           alt={post.title}
-          className="h-48 w-full rounded-t-lg object-cover"
+          className="h-48 sm:h-64 w-full rounded-t-lg object-cover"
         />
```

Comment:
Why are we making `h-64` for everything equal to and above the tablet devices? I assume we might have missed here, it should have been sm: h-48 and for the mobile h-full or something which suits here ... That should solve this, because if you observe carefully, the all blogs post card looks a bit stretched.

Reply:
Yes, it is supposed to e sm: h-48! Great observation! Thank you